CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ The Product Changelog at **[matomo.org/changelog](https://matomo.org/changelog)*
 
 * When requesting goals for multiple sites at once using `Goals.getGoals`, the result will no longer be indexed by `idgoal`. Requesting the goals for a single site will still return them indexed by `idgoal`.
 
+### New commands
+
+* New command `core:reset-invalidations` allows administrators to reset stuck invalidations that are incorrectly marked as "in progress".
```

Comment:
We don't know for sure if they are marked incorrectly, right? They might be running, so it's up to the admin to make the call after what time or for which host the invalidations could be considered as stuck and not running anymore. Should we maybe add more info around that?

```suggestion
* New command `core:reset-invalidations` allows administrators to reset stuck invalidations that might be incorrectly marked as "in progress".
```

Reply:
The command has a detailed description itself, so I wouldn't add too many details to the changelog.